.github/workflows/hls.yml

```diff
@@ -55,9 +55,15 @@ jobs: # A workflow run is made up of one or more jobs that can run sequentially
       with:
         name: TC
         path: project/trackletCalculator/solution1/syn/report/TrackletCalculator_L1L2G_csynth.rpt
-# Does not work right now
-#    - name: TE # Script name
-#      run: |
-#        cd project
-#        source $VIVADO_PATH_SH
-#        vivado_hls -f "script_TE.tcl"
+    - name: TE # Script name
+      run: |
+        cd emData/TE
+        ./download.sh
```

Comment:
do we need this? I think the goal is to have the repo and CI use the few data files that are already in the repo.

Reply:
The TE HLS script does not work without it.
@aehart Can we do the TE without the download script?